var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -17,6 +17,7 @@ class PyH5py(PythonPackage):
     import_modules = ['h5py', 'h5py._hl']
 
     version('master', branch='master')
+    version('3.1.0', sha256='1e2516f190652beedcb8c7acfa1c6fa92d99b42331cbef5e5c7ec2d65b0fc3c2')
```

Comment:
In upstream `spack/develop` there is more stuff related to v3, not sure if we want to ignore those:
`git diff spack/develop -- var/spack/repos/builtin/packages/py-h5py/package.py`

Reply:
I was about to ask actually... I am afraid to break everything if I push a new version.

Reply:
I would pull in the remaining changes, too. They seem mostly build-setup stuff, and would probably be good to have.